src/kirby/components/modal/modal-wrapper/modal-wrapper.component.tns.ts

```diff
@@ -66,4 +88,53 @@ export class ModalWrapperComponent extends ContentView {
     }
     // TODO: by default iOS slides up, however it slides together with the dimmed background, hence needs fixing
   }
+
+  getStatusBarHeight() {
+    var result = 0;
+    if (application.android) {
+      var resourceId = application.android.context
+        .getResources()
+        .getIdentifier('status_bar_height', 'dimen', 'android');
+      if (resourceId) {
+        const screenScale = screen.mainScreen.scale;
+        const pixelValue = application.android.context.getResources().getDimension(resourceId);
+        result = pixelValue / screenScale;
```

Comment:
And you are sure, that `screenScale` never can be 0 ?

Reply:
Pretty sure, it's what we already use for cards